blueocean-dashboard/src/main/js/util/Logger.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * A logger that provides links back to the actual log message source in chrome debug tools
```

Comment:
What is wrong with ` import { logging } from '@jenkins-cd/blueocean-core-js';` why yet-another logger?